projects/habitat_ovmm/scripts/gen_configs.py

```diff
@@ -0,0 +1,68 @@
+import os
+import time
+
+import yaml
+
+base_config_path = "projects/habitat_ovmm/configs/agent/hssd_eval.yaml"
+
+# Read in base config
```

Comment:
We should document how to use this script; + script execution should be guarded by if __name__ == 'main'

Reply:
Added the documentation already [here](https://github.com/facebookresearch/home-robot/blob/fadf0cf9aaebb5c095def14464118515d0300c78/src/home_robot_sim/README.md#run-evaluation).